modules/adagioBidAdapter.js

```diff
@@ -561,6 +605,21 @@ function isRendererPreferredFromPublisher(bidRequest) {
   );
 }
 
+/**
+ *
+ * @param {object} adagioStorage
+ * @returns {boolean}
+ */
+function isNewSession(adagioStorage) {
+  const now = Date.now();
+  const { lastActivityTime, vwSmplg } = utils.deepAccess(adagioStorage, 'session', {});
```

Comment:
Hi @musikele, thanks for your deep review!
Actually you see `navigation.session` because our external script has not been deployed yet and use the old schema.